helm/alfresco-content-services/README.md

```diff
@@ -0,0 +1,62 @@
+# Alfresco Content Services (ACS) Helm Chart
+
+Alfresco Content Services is an Enterprise Content Management (ECM) system that is used for document and case management, project collaboration, web content publishing, and compliant records management.  The flexible compute, storage, and database services that Kubernetes offers make it an ideal platform for Alfresco Content Services. This helm chart presents an enterprise-grade Alfresco Content Services configuration that you can adapt to virtually any scenario, and scale up, down, or out, depending on your use case.
+
+To use, add the `http://kubernetes-charts.alfresco.com/incubator` to your helm repository.
+```console
+helm repo add alfresco-incubator http://kubernetes-charts.alfresco.com/incubator
+```
+
+## To install the ACS cluster
+
+```console
+$ helm install alfresco-incubator/alfresco-content-services
+```
+
+## Introduction
+
+This chart bootstraps an ACS deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+  - Kubernetes 1.4+ with Beta APIs enabled
+
+## Installing the Chart
+
+To install the chart with the release name `my-acs`:
+
+```console
+# Alfresco Admin password should be encoded in MD5 Hash
+export ALF_ADMIN_PWD=$(printf %s 'MyAdminPwd' | iconv -t UTF-16LE | openssl md4 | awk '{ print $1}')
+
+# Alfresco Database (Postgresql) password
+export ALF_DB_PWD='MyDbPwd'
+
+$ helm install --name my-acs alfresco-incubator/alfresco-content-services \
+               --set repository.adminPassword="$ALF_ADMIN_PWD" \
+               --set postgresql.postgresPassword="$ALF_DB_PWD"
+```
+
+The command deploys ACS Cluster on the Kubernetes cluster in the default configuration (but with your chosen Alfresco administrator & database passwords). The [configuration](#configuration) section lists the parameters that can be configured during installation.
+
+> **Tip**: List all releases using `helm list`
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-acs` deployment:
+
+```console
+$ helm delete my-acs
```

Comment:
I would add '-- purge' parameter, as that releases the name my-acs and can be reused - that is a global name.